### PR TITLE
fix(sweeps): correct launch sweep author to personal username

### PR DESCRIPTION
**Pull request by gtarpenning**
Prod defaults to the entity passed in, which in many cases is the team entity. Update viewer api call to get username, and use that if available.

### CODE REVIEW

File: wandb/cli/cli.py [hunk 1]
```diff
@@ -949,6 +949,9 @@ def launch_sweep(
         wandb.termerror("A project must be configured when using launch")
         return
 
+    # get personal username, not team name or service account, default entity
```
Comment (KyleGoyette) [inline wandb/cli/cli.py:952]: Shouldn't this say `or default entity`?

**Review by KyleGoyette — APPROVED** [review]

File: wandb/cli/cli.py (hunk 1 above)
Comment (gtarpenning) [reply wandb/cli/cli.py:952]: damn talk about a nit

Comment (KyleGoyette) [reply wandb/cli/cli.py]: Should have prefixed with ULTRA nit 😛

**Review by kptkin — APPROVED** [review]
lgtm

**Pull request merged**